.circleci/config-2_1.yml

```diff
@@ -655,6 +662,21 @@ jobs:
           stop_on_failure: ${REPEATED_TESTS_STOP_ON_FAILURE}
           count: ${REPEATED_UPGRADE_DTESTS_COUNT}
 
+  j8_offheap_dtest:
+    <<: *j8_par_executor
+    steps:
+      - attach_workspace:
+          at: /home/cassandra
+      - log_environment
+      - clone_dtest
+      - create_venv
+      - create_dtest_containers:
+          file_tag: j8_with_vnodes
+          run_dtests_extra_args: "--use-vnodes --use-off-heap-memtables --skip-resource-intensive-tests --pytest-options '-k not cql'"
```

Comment:
To be honest, I am not sure where this came from - `--pytest-options '-k not cql'"`

Reply:
This is copy/pasta from the j8_with_vnodes template I was using. Will remove

Reply:
I honestly don't know how that got there. I'll remove it

Reply:
Got it, cqlsh tests in 4.0+ branches are separate jobs -k is a pytest option and we filter cql and non cql tests using it. But not in earlier branches where we have single dtest job. You will need it for the next branches 
https://docs.pytest.org/en/latest/how-to/usage.html
They say: "pytest -k "MyClass and not method"
This will run tests which contain names that match the given string expression (case-insensitive), which can include Python operators that use filenames, class names and function names as variables. The example above will run TestMyClass.test_something but not TestMyClass.test_method_simple."